projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -746,8 +746,14 @@ function wpcom_dismiss_removed_calypso_screen_notice() {
 						'extra'   => wp_json_encode( $preferences ),
```

Comment:
Can we add more information about the user (e.g. user_id) and site (e.g. site_id)? This will help us during debugging.

Reply:
Sure why not, I don't think it was a reason to block this PR since we didn't need this info before.

Reply:
Oh, yeah, I forgot to mention it's not blocking :P